data/DataType/short_cpp.yaml

```diff
@@ -1,4 +0,0 @@
-keyword: short
```

Comment:
I think you meant to rename this , not delete it.

Reply:
https://github.com/coala/coAST/pull/44#discussion_r232297046

Reply:
It shows it deleted since short_c.yaml has been merged via #46 .